src/main.ts

```diff
@@ -1,4 +1,40 @@
 import pino from 'pino';
+const logger = pino({
+  level: process.env.LOG_LEVEL || 'info',
+});
+
+import apm from 'elastic-apm-node';
+/* istanbul ignore next */
+{
+  const isApmEnabled = ['yes', 'true'].includes((process.env.ENABLE_APM || '').toLowerCase());
+
+  const apmServiceName = process.env.APM_SERVICE_NAME || 'paas-admin-local';
+
+  const apmSecretToken = process.env.APM_SECRET_TOKEN;
+  if (isApmEnabled && !apmSecretToken) {
+    throw new Error('APM_SECRET_TOKEN is a required environment variable');
+  }
+
+  const apmServerUrl = process.env.APM_SERVER_URL;
+  if (isApmEnabled && !apmServerUrl) {
+    throw new Error('APM_SERVER_URL is a required environment variable');
+  }
+
+  if (isApmEnabled) {
+    logger.info({
+      message: `Starting apm with name ${apmServiceName} -> ${apmServerUrl}`,
+    });
+  }
+
+  apm.start({
```

Comment:
Should this be in the `if (isApmEnabled)` block? or will it just do nothing if it's called with an empty URL

Reply:
There is an `active` key in the object passed to start, so other bits of the code where we talk to `apm` don't have to be wrapped in an if statement